src/components/App.js

```diff
@@ -27,7 +29,9 @@ function App() {
               <Route path="/signup" component={Signup} />
               <Route path="/login" component={Login} />
               <Route path="/forgot-password" component={ForgotPassword} />
-
+              <Route path="/events" component={Events} />
+              <Route path="/create-event" component={CreateEvent} />
+              <Route path="/myprofile" component={ProfileDashboard} /> 
```

Comment:
1. Change this component from `Route` to `PrivateRoute`
2. Change the endpoint from `myprofile` to `my-profile`. Make sure to update all references.